Mathlib/Algebra/Order/Nonneg/Field.lean

```diff
@@ -94,7 +94,13 @@ instance instNNRatSMul : SMul ℚ≥0 {x : α // 0 ≤ x} where
     (⟨q • a, by rw [NNRat.smul_def]; exact mul_nonneg q.cast_nonneg ha⟩ : {x : α // 0 ≤ x}) =
       q • a := rfl
 
-instance linearOrderedSemifield : LinearOrderedSemifield { x : α // 0 ≤ x } := fast_instance%
+instance semifield : Semifield { x : α // 0 ≤ x } :=
```

Comment:
Why no `fast_instance%` here? I should be clear that I don't know what this does, I'm just noting that it's missing here but is present for `linearOrderedSemifield`.